package.json

```diff
@@ -20,112 +20,104 @@
     "lint": "time npm-run-all --parallel --print-label --continue-on-error lint:*",
     "lint:css": "time stylelint src/**/*.scss",
     "lint:js": "time yarn run eslint:base server/ src/ support/ webpack/ jest.config.ts",
+    "prepare": "husky install",
     "start": "yarn run express:production",
     "test": "jest",
     "types:check": "npx tsc --noEmit --project tsconfig.json && npx tsc --noEmit --project server/tsconfig.json",
     "webpack:dev-server": "webpack serve --config=webpack/env.development.ts",
     "webpack:production": "webpack --config=webpack/env.production.ts",
     "webpack:stats": "yarn run webpack:production --env stats"
   },
-  "husky": {
-    "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-commit": "lint-staged"
-    }
-  },
   "lint-staged": {
-    "*.{js,mjs,mts}": "yarn run eslint:base",
+    "*.{js,ts,tsx,mts}": "yarn run eslint:base",
```

Comment:
💡 Unrelated; fixing an issue where we weren't targeting `ts` and `tsx` files for linting.